core/src/main/java/com/datastax/dse/driver/internal/core/graph/ContinuousGraphRequestHandler.java

```diff
@@ -99,7 +99,7 @@ protected Duration getGlobalTimeout() {
   @NonNull
   @Override
   protected Duration getPageTimeout(int pageNumber) {
-    return Duration.ZERO;
+    return this.globalTimeout;
```

Comment:
I have concerns about the full effect of this change. Currently, a value of 0 means that the driver will wait forever for a response to successive page requests (the first page timeout is controlled by the globalTimeout).

I changed the return value here to be the globalTimeout value as the handler doesn't have a separate page timeouts (like [ContinuousCqlRequestHandler](https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/dse/driver/internal/core/cql/continuous/ContinuousCqlRequestHandler.java#L51-L52).

The test that fails is [here](https://github.com/datastax/java-driver/pull/1431/files#diff-c96ea9340f349d35ca1642eb4f50c76aR490), as it expects the second page request to timeout while CCM has been paused. But with this method returning a Duration of 0, it never times out the request. Instead, we get a Heartbeat timeout 30 seconds later. This also generally causes many of the DatatProvider tests in that class to fail as more queries are attempted before the session has been fully re-established after CMM is resumed, which causes the session to die with an AllNodesFailed exception.

I'm not sure if this is the best solution, as maybe GraphPaging needs to support setting a timeout for `otherPages` like the ContiuousCqlRequestHandler does now.